packages/babel-parser/test/helpers/run-fixture-tests.js

```diff
@@ -81,7 +84,9 @@ function runAutogeneratedParseTests(
       original: index === 0,
     }))
     .forEach(test =>
-      testFn(test.title, () => runParseTest(parse, test, onlyCompareErrors)),
+      testFn(test.title, () =>
+        runParseTest(parse, test, onlyCompareErrors, ignoreLocations),
```

Comment:
```suggestion
        runParseTest(parse, test, onlyCompareErrors, test.adjust !== false),
```

since `test.adjust` is `false` for the original test, and a function for the fuzzed ones